content/yaml-publishing/publish-release-notes.md

```diff
@@ -11,7 +11,8 @@ Release notes can be published to:
 
 * **email**. The release notes will be included in the publishing email of a successful build if you have publishing to email configured in the `publishing` section of your workflow.
 * **Slack**. The release notes will be included in the Slack notification of a successful build if you have publishing to Slack configured in the `publishing` section of your workflow.
-* **App Store Connect**. The release notes will be published to the What to Test field in TestFlight if you have publishing to App Store Connect configured in the `publishing` section of your workflow. Note that release notes are uploaded in the [post-processing step](/yaml-publishing/distribution#post-processing-of-app-store-connect-distribution).
+* **App Store Connect**. The release notes will be published as "What to Test" field under Test Details section in TestFlight if you have publishing to App Store Connect configured in the `publishing` section of your workflow. In case App Store review submission is enabled, then the release notes will also be used to fill in the "What's New" field under Version Information section in App Store.
```

Comment:
```suggestion
* **App Store Connect**. The release notes will be published to the "What to Test" field in the Test Details section in TestFlight if you have publishing to App Store Connect configured in the `publishing` section of your workflow. In case App Store review submission is enabled, then the release notes will be also used to fill in the "What's New" field in the Version Information section in App Store.
```